datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/JavaTimeFeature.java

```diff
@@ -17,7 +17,17 @@ public enum JavaTimeFeature implements JacksonFeature
      * Default setting is enabled, for backwards-compatibility with
      * Jackson 2.15.
      */
-    NORMALIZE_DESERIALIZED_ZONE_ID(true);
+    NORMALIZE_DESERIALIZED_ZONE_ID(true),
+
+    /**
+     * Feature that controls whether numeric strings are interpreted as numeric
+     * timestamps (enabled) nor not (disabled) in addition to an explicitly
+     * defined pattern.
+     * <p>
+     * Note that when a pattern is not explicitly defined numeric strings are
+     * interpreted as a numeric timestamp.
+     */
+    READ_NUMERIC_STRINGS_AS_DATE_TIMESTAMP(false);
```

Comment:
When it comes to `ALWAYS_ READ_NUMERIC_STRINGS_AS_DATE_TIMESTAMP` probably works.

The `ALWAYS` implies that even when disabled it happens some times.

There still is the implication that _any_ numeric string is read as a timestamp.

So then we get `ALWAYS_READ_DATE_TIMESTAMPS_FROM_NUMERIC_STRING`?

But your choice really. I'm not terribly attached to any of these.

Reply:
:-)

Naming is hard!

So maybe something like

     ALWAYS_ALLOW_STRINGIFIED_TIMESTAMPS

would be good, to indicate that timestamps-as-strings are allowed, without claiming all JSON Strings must be timestamps (this will still only augment possibilities).

Let's try that one, get everything else straightened out, and this can still make it in 2.16.0.